minio/helpers.py

```diff
@@ -696,3 +700,18 @@ def is_supported_header(key):
 # returns true if header is a storage class header
 def is_storageclass_header(key):
     return key.lower() == "x-amz-storage-class"
+
+ # return the input type if valid i.e. csv, json,  parquet, par 
+ # or raise an InvalidArgumentError 
+def is_valid_input_type(object_name):
+    if  object_name.__contains__('.csv')  |  object_name.__contains__('.CSV') :
```

Comment:
we could use `object_name.lower().endswith(ext)` here

Reply:
We should avoid using `__functions__` it is better to use higher-level functions - also no reason to add this validation. The server is supposed to validate this properly, any file without the extension with proper content-type works fine. So this is a superfluous check which should be removed.